### ISSUE TITLE
Double line breaks are stripped out

### ISSUE THREAD

**Issue by danielallsopp** [ZhgChgLi/ZMarkupParser#40]
In the following HTML example:
`One<br /><b>Two</b><br /><i>Three</i><br /><i><b>Four</b></i><br /><br />New <b>Paragraph</b>!`

should render with a line break after the 'Four' and before the 'New Paragraph' is rendered. However, I've noticed when the HTML string is parsed, the second line break is stripped out, so the render ends up like this:

![incorrect](https://github.com/ZhgChgLi/ZMarkupParser/assets/6536989/5c998292-2a52-44be-be30-25302cc47677)

instead of this:

![correct](https://github.com/ZhgChgLi/ZMarkupParser/assets/6536989/a92e2844-19fa-4143-8148-d35c2299f792)

Can this functionality be switched off, or is there a bug present?

EDIT: I've found this function in the source code which seems to the be the culprit:

```
func reduceBreaklineInResultNSAttributedString(_ attributedString: NSAttributedString) -> NSAttributedString
```

but I can't find anyway of switching it off, or overriding the functionality.

**Comment by charrondev** [comment]
@danielallsopp You can see in my recent PR here I added a policy to modify line break behaviour. https://github.com/ZhgChgLi/ZMarkupParser/pull/37

For me I found the behaviour was adding more line breaks that I preferred. Maybe there could be another line break policy added that makes no changes to the built-in line breaks. (call it raw or something like that?).

**Comment by www-r3m1x** [comment]
@zhgchgli0718 This bug not fixed in 1.7.0

**Comment by zhgchgli0718** [comment]
![image](https://github.com/ZhgChgLi/ZMarkupParser/assets/33706588/bf51108e-399e-4ece-b216-804375967925)
In the latest version 1.8.0, the issues have been addressed.
Thank you for your patience.
